src/core/components/operation-tag.jsx

```diff
@@ -72,48 +72,49 @@ export default class OperationTag extends React.Component {
 
         <h3
           onClick={() => layoutActions.show(isShownKey, !showTag)}
-          className={!tagDescription ? "opblock-tag no-desc" : "opblock-tag" }
+          className={!tagDescription ? "opblock-tag no-desc" : "opblock-tag"}
           id={isShownKey.map(v => escapeDeepLinkPath(v)).join("-")}
           data-tag={tag}
           data-is-open={showTag}
-          >
+        >
           <DeepLink
             enabled={isDeepLinkingEnabled}
             isShown={showTag}
             path={createDeepLinkPath(tag)}
             text={tag} />
-          { !tagDescription ? <small></small> :
+          {!tagDescription ? <small></small> :
             <small>
-                <Markdown source={tagDescription} />
-              </small>
-            }
+              <Markdown source={tagDescription} />
+            </small>
+          }
 
+          {!tagExternalDocsDescription ? null :
             <div>
```

Comment:
Let's add a className="info__externaldocs" to this div. This would enable easier Cypress assertion if the div exists. Also helps identify that this may refactored in the future into its own component.

Reply:
Thanks, that will make testing much cleaner!